client/components/segmentation-survey/index.tsx

```diff
@@ -1,13 +1,14 @@
 import React, { useCallback } from 'react';
 import DocumentHead from 'calypso/components/data/document-head';
 import SurveyContainer from 'calypso/components/survey-container';
-import { Question } from 'calypso/components/survey-container/types';
+import { Question, QuestionConfiguration } from 'calypso/components/survey-container/types';
 import {
 	useCachedAnswers,
 	useSaveAnswersMutation,
 	useSurveyStructureQuery,
 } from 'calypso/data/segmentaton-survey';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { QuestionComponentMap } from '../survey-container/components/question-step-mapping';
```

Comment:
Can this be in the 'calypso/components...' format? :)

Reply:
Sure, I'll fix that, thanks!